test/suite/test_tiered19.py

```diff
@@ -413,3 +420,4 @@ def test_ss_azure_file_system(self):
 
         # Test that azure storage source terminate succeeds.
         self.assertEqual(ss.terminate(session), 0)
+        self.ignoreStdoutPatternIfExists('HTTP status code')
```

Comment:
Should this include the code? I could see this returning (e.g.) 404 which would be bad, but we'd ignore it here.

Reply:
This one was an interesting one - I see your point which I totally overlooked when first writing this. To avoid including the codes (200, 201, and 206) i've simplified it to that. Should I just add the 3 lines after?

Reply:
All of the 2xx codes are "success" in one form or another, so I think you could include the "2" and call it done.

Reply:
thanks i've added this now to the line https://github.com/wiredtiger/wiredtiger/pull/8834/commits/fc36ee81bd7168a41031c899e142018462d4393e